src/components/App/AppData.js

```diff
@@ -17,6 +18,15 @@ function AppData() {
     mySitesData,
   } = useMySites()
 
+  const sites = useSites()
```

Comment:
Performing the data retrieval and manipulation at this level will result in it being performed every time the application loads regardless of what page the user is on. Could we perform it locally? Perhaps in an `FVAppsData` layer?